### PR TITLE
Update max rate-limiting metadata

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What

Updating max rate-limit metadata for sources Github, Sentry, Zendesk Support

[Github:](https://docs.github.com/en/graphql/overview/rate-limits-and-node-limits-for-the-graphql-api) Per-hour window listed in docs.
[Sentry:](https://docs.sentry.io/api/ratelimits/) Docs do not explicitly list retry windows, and point to checking retry headers to determine these values. All endpoints returned a 1-minute retry window in tests.
[Zendesk Support:](https://developer.zendesk.com/api-reference/introduction/rate-limits/) Per-minute values listed in docs.

### CODE REVIEW

**Review by alafanechere — APPROVED** [review]

File: airbyte-integrations/connectors/source-sentry/metadata.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ data:
   githubIssueLabel: source-sentry
   icon: sentry.svg
   license: MIT
-  maxSecondsBetweenMessages: 64800
+  maxSecondsBetweenMessages: 60
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-sentry/metadata.yaml:19]: Checking the [Sentry API doc](https://docs.sentry.io/api/ratelimits/), it seems like the rate limit window is a second: `We restrict both how frequently a request is made (requests per second rate limit) <...>`. What is the reasoning for having 60 seconds here?

**Review by maxi297 — APPROVED** [review]
Just one question about Sentry but I'll provide conditional approval so that I'm not blocking the release

File: airbyte-integrations/connectors/source-sentry/metadata.yaml (hunk 1 above)
Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-sentry/metadata.yaml:19]: I was originally a bit confused by this as well, but when testing and viewing the response headers, the `x-sentry-rate-limit-reset` was being set to the next full minute from the request (so a request sent at `12:00:02` will return a reset time of `12:01:00`, as will a request from `12:00:36`). Worth noting that I ran some more tests and some of the streams do seem to have smaller windows, but the `events` stream in particular consistently sets this value to the next full minute.

Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-sentry/metadata.yaml:23]: Sure thing, just added a comment for clarity 👍 I agree this value may actually be higher given it's not as set in stone as their docs would suggest, but my understanding is also that at the moment we don't actually do anything with this value unless it's at least 5400 seconds (or 90 minutes). Is that incorrect?

**Pull request merged**